### PR TITLE
User stats per day

### PR DESCRIPTION
**Pull request by MichalGniadek**
Example use
```
const result = await fetch("http://127.0.0.1:8000/user-statistics/", {
  method: "POST",
  body: JSON.stringify({
    game: "memory",
    start: new Date(2020).toISOString(),
    end: new Date().toISOString(),
  }),
  headers: {
    Authorization: `Token ${token}`,
  },
});
```

### CODE REVIEW

File: api/views.py [hunk 1]
```diff
@@ -234,6 +234,38 @@ def get_statistics(request):
...
+        body["end"],
+    )
+
+    if not game or not start or not end:
+        return HttpResponseBadRequest(
+            "Body must contains 'game', 'start' and 'end'"
+        )
+
+    objects = None
+    if game == "memory":
```
Comment (mSkarbinska) [inline api/views.py:254]: can we use enum here? I believe enum is already defined in models or helpers

File: api/views.py [hunk 2]
```diff
@@ -234,6 +234,38 @@ def get_statistics(request):
 
     return Response(data=list(result))
 
+@api_view(["POST"])
```
Comment (mSkarbinska) [inline api/views.py:237]: why POST when the method is called `get_user_statistics` ?

Comment (MichalGniadek) [reply api/views.py:237]: We need to use POST to be able to pass a body with request details.

Comment (mSkarbinska) [reply api/views.py:237]: makes sense

File: api/views.py (hunk 1 above)
Comment (MichalGniadek) [reply api/views.py:254]: I don't think we have a proper enum for this yet. We should also change it in other places. I will create an issue to do this later everywhere.

**Comment by mSkarbinska** [comment]
lgtm

**Pull request merged**